core/server/api/index.js

```diff
@@ -291,7 +291,7 @@ http = function http(apiMethod) {
         return apiMethod(object, options).tap(function onSuccess(response) {
```

Comment:
> function onSuccess(response) {

What about this named fn? 🤔 

---

Named functions in Promise chains **can** be helpful for reading error stacks, but **only** if the error stack does not not show proper/correct line numbers of the target file?

I think we need some more time to figure out when to use arrow functions and when not.